gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-client/src/test/java/org/apache/gobblin/service/FlowConfigTest.java

```diff
@@ -220,6 +221,7 @@ public void testUpdate() throws Exception {
     //Assert.assertEquals(flowConfig.getProperties().size(), 2);
     Assert.assertEquals(retrievedFlowConfig.getProperties().get("param1"), "value1b");
     Assert.assertEquals(retrievedFlowConfig.getProperties().get("param2"), "value2b");
+    Assert.assertEquals(retrievedFlowConfig.getProperties().get(RequesterService.REQUESTER_LIST), RequesterService.serialize(new ArrayList<>()));
```

Comment:
Hmm, so there is no way to have a legitimate requester in tests?

Reply:
Not without manually setting it, I don't think it matters though, empty list is still fine as a test that it's there.